src/modules/todos/controllers.js

```diff
@@ -1,144 +1,154 @@
-import { todos} from '../database/database.js';
-import AppException from '../../utils/exceptions/AppException.js';
-import { v4 } from 'uuid';
+import { v4 } from "uuid";
 
+import pool from "../../database/database.js";
+import AppException from "../../utils/exceptions/AppException.js";
+import queries from "./queries.js";
 
-export function getTodos(req, res) {
-	res.json(todos.filter((todo) => todo.userId === req.userId));
-};
+export async function getTodos(req, res, next) {
+	try {
+		const allTodos = await pool.query(queries.allTodos, [req.userId]);
```

Comment:
I like that you took the initiative to move all the queries into a separate file. This works, but it can be even better. We will revisit it.